### ISSUE TITLE
problems with 'temporary eclosion muscle' terms [sf#30]

### ISSUE THREAD

**Issue by mmc46** [mmc46/drosophila-anatomy-developmental-ontology#30]
**Reported by gm119 on 2014-04-02 13:13 UTC**
Hi,

I think that there are a number of problems with the terms that sit under 'temporary eclosion muscle' in the anatomy ontology.

I initially was writing about just the first item listed below, but found the others while trying to write the ticket and started looking at the section in detail.

1. location/existence of the following term:

[Term]
id: FBbt:00010641
name: abdominal 1 temporary eclosion muscle DA2
def: "Temporary eclosion muscle of adult abdominal segment 1 that develops from abdominal 1 dorsal acute muscle 2 of the larva by a process of remodelling." [FlyBase:FBrf0052405, FlyBase:FBrf0201818]
synonym: "A1 DIOM2" EXACT [FlyBase:FBrf0052405]
is_a: FBbt:00003447 ! skeletal muscle of abdomen
is_a: FBbt:00016665 ! temporary eclosion muscle
relationship: part_of FBbt:00003025 ! adult abdominal segment 1

It sits as a child of temporary eclosion muscle, but other similar terms are one level down, under 'abdominal temporary eclosion muscle DA' (FBbt:00007671), so I initially thought that FBbt:00010641 might need to go under 'abdominal temporary eclosion muscle DA':

[Term]
id: FBbt:00007671
name: abdominal temporary eclosion muscle DA
def: "Temporary eclosion muscle of the adult abdomen that develops from an A1-7 dorsal acute muscle." [FlyBase:FBrf0052405, FlyBase:FBrf0201818]
synonym: "TDIOM" EXACT [FlyBase:FBrf0201818]
synonym: "temporary dorsal internal oblique muscle" EXACT [FlyBase:FBrf0201818]
is_a: FBbt:00003447 ! skeletal muscle of abdomen
is_a: FBbt:00016665 ! temporary eclosion muscle
relationship: develops_from FBbt:00000590 ! A1-7 dorsal acute muscle
relationship: part_of FBbt:00003023 ! adult abdomen

Its possible that there is something special about the temporary muscle of A1 that is derived from DA2, but the definition of 'abdominal temporary eclosion muscle DA' is just defined as any temporary muscle that comes from an A1-7 DA muscle, so it looked wrong to me !

I just looked in the ontology to check that "abdominal 1 dorsal acute muscle 2" is a A1-7 DA muscle and found that it is an obsolete term:

[Term]
id: FBbt:00000641
name: abdominal 1 dorsal acute muscle 2
comment: This muscle is not present in A1 (Bate, 1993).
subset: cur
synonym: "A1 DA2" EXACT [FlyBase:FBrf0064793, FlyBase:FBrf0089570]
is_obsolete: true
consider: FBbt:00000592

as apparently A1 DA2 does not exist !

However, the definition of 'abdominal 1 temporary eclosion muscle DA2' (FBbt:00010641) says it is derived from A1 DA2 (?!)

So maybe FBbt:00010641 needs removing from the ontology, or the definition needs changing ?

If it is in the right place, it would be good to have a comment explaining why, so that if we stumble across it in the future, we don't go through this again (!)

2. from looking at  FBrf0201818 (which is listed as a reference for this set of terms), which says on pg. 387:

"Persistent larval muscles (DIOMs) are remodeled into the temporary dorsal eclosion muscles (TDIOMs). In the 3 days following PPT, four major processes take place to change muscle morphology (Fig. 5E, Movies S1 and S5). (1) The DIOM muscle fibers shrink to about one quarter their original diameter from about 40 μm to 10 μm. (2) They translocate and rotate into an orientation that is more parallel to the midline. (3) Some break apart into two fragments. For instance, DIOM of abdominal segment 3 splits to become TDIOM2 of segment 2 and TDIOM1 of segment 3."

(in this case they arae using DIOM for larval muscles that are remodelled to form the temporary eclosion ones, but FBrf0052405 uses DIOM for the temporary eclosion ones, so its a bit confusing !!)

Its part 3) that may not be reflected in the ontology terms we have - from what it says above, I think that there should be two temporary eclosion muscle terms that say they are derived from a larval abdominal segment 3 muscle, but we don't.

3. Some of the definitions do not match the numbers in the terms - I'm not sure whether or not this is correct, but if e.g. 'abdominal 1' is meant to mean 'adult abdominal segment 1', and 'abdominal 2' is meant to mean 'adult abdominal segment 2' then there is a mismatch between term and name in the following cases.  If the mismatch is deliberate (e.g. if its another case of larval muscles splitting into muscles which end up in different adult segments), it might be good to add a comment explaining what is going on.

??def wrong if relationship 'part_of FBbt:00003026 ! adult abdominal segment 2' is correct??
[Term]
id: FBbt:00010710
name: abdominal 2 temporary eclosion muscle LL1
def: "Temporary eclosion muscle of adult abdominal segment 1 that develops from abdominal 2 lateral longitudinal muscle 1 of the larva by a process of remodelling." [FlyBase:FBrf0052405, FlyBase:FBrf0201818]
is_a: FBbt:00007672 ! abdominal temporary eclosion muscle LL1
relationship: develops_from FBbt:00000710 ! abdominal 2 lateral longitudinal muscle 1
relationship: part_of FBbt:00003026 ! adult abdominal segment 2

??def wrong if relationship 'part_of FBbt:00003026 ! adult abdominal segment 2' is correct??[Term]
id: FBbt:00010712
name: abdominal 2 temporary eclosion muscle VL1
def: "Temporary eclosion muscle of adult abdominal segment 1 that develops from abdominal 1 ventral longitudinal muscle 1 of the larva by a process of remodelling." [FlyBase:FBrf0052405, FlyBase:FBrf0201818]
is_a: FBbt:00007673 ! abdominal temporary eclosion muscle VL
relationship: develops_from FBbt:00000712 ! abdominal 2 ventral longitudinal muscle 1
relationship: part_of FBbt:00003026 ! adult abdominal segment 2

??if def is correct and really does develop from A1 SBM, might be nice to add comment clarifying that, so we don't think its a mistake in the future??
[Term]
id: FBbt:00010716
name: abdominal 2 temporary eclosion muscle SBM
def: "Temporary eclosion muscle of adult abdominal segment 2 that develops from abdominal 1 segment border muscle of the larva by a process of remodelling." [FlyBase:FBrf0052405, FlyBase:FBrf0201818]
is_a: FBbt:00007674 ! abdominal temporary eclosion muscle SBM
relationship: develops_from FBbt:00000716 ! abdominal 2 segment border muscle
relationship: part_of FBbt:00003026 ! adult abdominal segment 2

??def wrong if relationship 'part_of FBbt:00003028 ! adult abdominal segment 4' is correct??
?? the develops_from is quite vague for this one, and doesn't match the def - if its deliberate, can there be a comment explaining why ??
[Term]
id: FBbt:00010775
name: abdominal 4 temporary eclosion muscle DA1
def: "Temporary eclosion muscle of adult abdominal segment 1 that develops from abdominal 1 dorsal acute muscle 1 of the larva by a process of remodelling." [FlyBase:FBrf0052405, FlyBase:FBrf0201818]
is_a: FBbt:00007671 ! abdominal temporary eclosion muscle DA
relationship: develops_from FBbt:00000591 ! A1-7 dorsal acute muscle 1
relationship: part_of FBbt:00003028 ! adult abdominal segment 4

??def wrong if relationship 'part_of FBbt:00003027 ! adult abdominal segment 3
' is correct or maybe this is one that moves segment between larva and adult??
[Term]
id: FBbt:00020761
name: abdominal 3 temporary eclosion muscle SBM
def: "Temporary eclosion muscle of adult abdominal segment 2 that develops from abdominal 3 segment border muscle of the larva by a process of remodelling." [FlyBase:FBrf0052405, FlyBase:FBrf0201818]
is_a: FBbt:00007674 ! abdominal temporary eclosion muscle SBM
relationship: develops_from FBbt:00000761 ! abdominal 3 segment border muscle
relationship: part_of FBbt:00003027 ! adult abdominal segment 3

**Comment by mmc46** [comment]
**Commented by mmc46 on 2014-05-05 15:56 UTC**

**Comment by mmc46** [comment]
**Commented by mmc46 on 2014-05-05 15:56 UTC**
Hi Gillian,

1. location/existence of the following term: abdominal 1 temporary eclosion muscle DA2
This was related to a mistake in obsoleting 'abdominal 1 dorsal acute muscle 2'. I've corrected the related terms to this one, and added back a relationship that makes 'abdominal 1 temporary eclosion muscle DA2' be a subclass of 'abdominal temporary eclosion muscle DA'.
2. possible new terms from FBrf0201818
"(3) Some break apart into two fragments. For instance, DIOM of abdominal segment 3 splits to become TDIOM2 of segment 2 and TDIOM1 of segment 3."
I have not added the terms for the muscles they say split because:
- that is not clear or labelled in the figure and movies they mention (Fig. 5E, Movies S1 and S5). It seems that there is a split in segments 2 and 3, but I can't find evidence that they move between segments. 
- it is not clear which DIOM muscles (DA) they mean. They present segment 3 as an example, implying it happens in others. Segment 3 contains 2 of the 3 DA muscles, in other segments there are 3 DA muscles.
- haven't found any other data in papers/books that mentions this. 

I did add a comment to 'abdominal temporary eclosion muscle DA' saying that FBrf0201818 suggests that some muscles split.

3. Some of the definitions do not match the numbers in the terms
I couldn't find any data to support the mismatch, so I think it was just a series of mistakes. I've corrected the definitions, relationships (when needed) and added the specific synonyms from FBrf0052405.
